omnibus/config/projects/agent.rb

```diff
@@ -321,6 +324,16 @@
     end
   }
 
+  raise_if_fips_symbol_not_found = Proc.new { |symbols|
+      count = symbols.scan("github.com/microsoft/go-crypto-winnative").count()
+      if count == 0
+        puts "Checking for FIPS symbols: '#{symbols}'"
+        raise FIPSSymbolsNotFound.new("Expected to find symbol 'cng' but no symbol was found.")
```

Comment:
is the `cng` the same as `github.com/microsoft/go-crypto-winnative`? Should we update the error message?

Reply:
There are tied but you're right here, I'll update it 👍